.rubocop.yml

```diff
@@ -148,6 +148,8 @@ Rails/AfterCommitOverride: # (new in 2.8)
   Enabled: true
 Rails/FindById: # (new in 2.7)
   Enabled: true
+Rails/HasAndBelongsToMany:
```

Comment:
Can you say more about the rationale to flip this one off?

Reply:
Because we don't need any additional attributes on the association: https://rails.rubystyle.guide/#has-many-through   rubocop-rails is being especially paternalistic with this one.